packages/core/tracing/src/composables/index.ts

```diff
@@ -0,0 +1,8 @@
+import useDurationFormatter from './useDurationFormatter'
+import useI18n from './useI18n'
+
+// All composables must be exported as part of the default object for Cypress test stubs
+export default {
+  useDurationFormatter,
+  useI18n,
+}
```

Comment:
Shall we just export some util functions instead of wrapping everything into composables? Composables should be related to reactivity and component lifecycles.